File: ViewModels/ViewModels/LanguageViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ViewModels.ViewModels
{
    public class LanguageViewModel : BaseViewModel
    {
        public string Name { get; set; }

        public string Code { get; set; }

        public Dictionary<string, string> Dictionary { get; set; }
    }
}

File: src/musicgo/src/shared/classes/LanguageViewModel.ts
import { Dictionary } from 'vue-router/types/router';
import { Guid } from "guid-typescript";
export class LanguageViewModel {
    public Id: string='00000000-0000-0000-0000-000000000000';
    public Name: string = '';
    public Code: string = '';
    public Dictionary: Dictionary<string> | null = null;
}
